### PR TITLE
Likes and comments

### PR DESCRIPTION
**Pull request by Patrisha-de-Boon**
Add the likes and comments api and continue to improve existing apis

### CODE REVIEW

File: socialdistribution_root/socialdistribution/utils.py [hunk 1]
```diff
@@ -12,4 +12,15 @@ def getAcceptedMediaType(request: HttpRequest):
...
     @staticmethod
     def serialize(serializer: ModelSerializer, request: HttpRequest):
-        return JSONRenderer().render(serializer.data, Utils.getAcceptedMediaType(request))
\ No newline at end of file
+        return JSONRenderer().render(serializer.data, Utils.getAcceptedMediaType(request))
+
+    
+    # Used for formatting and styling responses
+    @staticmethod
+    def compose_posts_dict(query_type, data):
```
Comment (wowikc) [inline socialdistribution_root/socialdistribution/utils.py:20]: we should probably give it a better name at some point if it is being used for all kind of stuff 😅 
Maybe like, compose response, or format response

**Review by wowikc — APPROVED** [review]
I think its looks fine besides permission comment on posts.
I was not able to test it though 👀

File: socialdistribution_root/socialdistribution/utils.py (hunk 1 above)
Comment (Patrisha-de-Boon) [reply socialdistribution_root/socialdistribution/utils.py:20]: Good point, I'll do that

File: socialdistribution_root/apis/posts/views.py [hunk 2]
```diff
@@ -105,8 +98,9 @@ def delete(self, request: HttpRequest, author_id: str, post_id: str, format=None
 
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+
 class posts(GenericAPIView):
-    permission_classes = [IsOwnerOrReadOnly]
+    # permission_classes = [IsOwnerOrReadOnly]
```
[the remark this answers is not in the document]
Comment (Patrisha-de-Boon) [reply socialdistribution_root/apis/posts/views.py:103]: Good catch, that appears to be a relic of previous testing. I'll uncomment that line.

**Pull request merged**